docs/detailed_test_config.rst

```diff
@@ -270,3 +270,15 @@ the location of the sandbox directory, the following option can be
 specified in ``$HOME/.teuthology.yaml``::
 
     test_path: <directory>
+
+Shaman options
+==============
+
+Shaman represents a project that is built by gitbuilder, which will
```

Comment:
we don't use gitbuilder anymore. jenkins.ceph.com is used for building packages.

Reply:
Fixed the doc, please take a look. Thanks.